libensemble/manager.py

```diff
@@ -433,8 +433,9 @@ def _update_state_on_worker_msg(self, persis_info, D_recv, w):
             else:
                 self._freeup_resources(w)
 
-        if 'persis_info' in D_recv and len(D_recv['persis_info']):
-            persis_info[w].update(D_recv['persis_info'])
+        if D_recv.get('persis_info') is not None:
```

Comment:
Remind me - why not just if D_recv.get('persis_info') instead of 436/437

Reply:
That's probably valid. The original work was to address how persis_info is handled by the manager if the user didn't specify one but the alloc tried packing it up anyway.